### ISSUE TITLE
HELP

### ISSUE THREAD

**Issue by CRAKI34** [gloglas/WifiDuckV2#2]
In the spacehun project there are two .Ino files, one for the Arduino pro micro and the other for the wemos d1 mini or esp8266, and in this only 1, I need help

**Comment by gloglas** [comment]
Hi, I modified only the arduino side of the project. Maybe you can find esp ino in older commits.

 But new spacehuhn WifiDuck is better. If I will find some time I will implement os detection there as well.

**Comment by CRAKI34** [comment]
I did not understand very well, do you mean that you only modified the file for the arduino pro micro?

**Comment by gloglas** [comment]
Yes. But you need to find same version. It is not compatible with the newest one.

**Comment by CRAKI34** [comment]
puedes compartirme la versión necesaria por favor

**Comment by CRAKI34** [comment]
you can share me the version necesary please

**Comment by gloglas** [comment]
https://github.com/spacehuhn/wifi_ducky/
